### ISSUE TITLE
github.com certificate validation

### ISSUE THREAD

**Issue by staabm** [KnpLabs/php-github-api#1028]
would it make sense, before calling endpoints of the github.com api (and before sending secret accesstokens over the wire) to validate the ssl certificate of the endpoint, so we are sure we are actually sending the secret data over to github.com and not another system which pretends to be github.com?

**Comment by acrobat** [comment]
Yes a docs entry would be indeed a good idea! (and a pr is welcome!) For actually implementing it in the library itself, I might not see it fit actually. First of all as you mentioned we abstracted away the actual httpclient so this can be added on application level wel you setup the httpclient with httplug. On the other hand this library can be used with an enterprise setup so validating github.com won't do the trick.

So all in all I'm more in favor of giving this a clear place in the docs so people who are looking for this kind of extra check are able to do it.
